src/components/scene/inspector.js

```diff
@@ -12,12 +12,35 @@ module.exports.Component = registerComponent('inspector', {
   },
 
   init: function () {
+    this.loadingMessage = 'Loading Inspector';
     this.onKeydown = bind(this.onKeydown, this);
     this.onMessage = bind(this.onMessage, this);
+    this.initOverlay();
     window.addEventListener('keydown', this.onKeydown);
     window.addEventListener('message', this.onMessage);
   },
 
+  initOverlay: function () {
+    this.overlayEl = document.createElement('div');
+    this.overlayEl.style.backgroundColor = '#222';
```

Comment:
Is not constructing the string to set multiple CSS attributes more cumbersome than just setting `style.XXX = YYY`